core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts

```diff
@@ -50,8 +54,8 @@ export class StateAndContent {
     public translation: string | string[],
     public dataFormat: string,
     public contentType: string,
-    public interactionId?: string,
-    public ruleType?: string
+    public interactionId: string | null,
+    public ruleType: string | null
```

Comment:
"translatableItem.interactionId, translatableItem.ruleType "  is assigned to a type string | null , so to remove the conflict i added null as a type to them

Reply:
I meant a comment in the file itself above these properties, you can take a look at docstrings from the backend to make the comment look more appropriate. Note: If a property is `null` in the frontend then it means that it can be `None` in the backend.

Reply:
Done.